app/models/user.rb

```diff
@@ -215,27 +201,21 @@ def set_default_flags
     self.welcome_email_sent = true if (is_developer? and !skip_welcome_email)
   end
 
-  def push_to_nutshell
-    if contact_name_changed? || email_changed? || approval_status_changed?
-      NutshellAdapter.new.push_user self
-    end
-  end
-
   def send_welcome_to_new_user_email
     Emailer.welcome_to_new_user(self).deliver_now
   end
 
   def audit_trail_user_added(site, source)
-    add_nutshell_note_for_user('added', 'to', site, source)
+    log_note_for_user('added', 'to', site, source)
   end
 
   def audit_trail_user_removed(site, source)
-    add_nutshell_note_for_user('removed', 'from', site, source)
+    log_note_for_user('removed', 'from', site, source)
   end
 
-  def add_nutshell_note_for_user(added_or_removed, to_or_from, site, source)
-    note = "#{source} #{added_or_removed} @[Contacts:#{self.nutshell_id}], #{self.email} #{to_or_from} @[Leads:#{site.nutshell_id}] #{site.display_name} [#{site.name}]."
-    NutshellAdapter.new.new_note(self, note)
+  def log_note_for_user(added_or_removed, to_or_from, site, source)
+    note = "#{source} #{added_or_removed} User #{id}, #{email}, #{to_or_from} Affiliate #{site.id}, #{site.display_name} [#{site.name}]."
```

Comment:
Maybe we can break this long line into 2 or 3 lines? its very long.

Reply:
can do 👍